src/extensions/surveys.tsx

```diff
@@ -701,7 +701,6 @@ export function FeedbackWidget({
 }): JSX.Element {
     const [showSurvey, setShowSurvey] = useState(false)
     const [styleOverrides, setStyle] = useState({})
-    const widgetRef = useRef<HTMLDivElement>(null)
```

Comment:
I ended up removing this as it's no longer needed. It muddies up the diff quite a bit, but have a look at the individual commits if you'd like to separate this change from the rest.